ci/Jenkinsfile.android

```diff
@@ -31,6 +31,11 @@ pipeline {
       description: 'Specify build type. Values: pr / e2e / nightly / release',
       defaultValue: 'pr',
     )
+    string(
+      name: 'ALLOWED_APK_SIZE',
+      description: 'Maximum allowed size of apk',
+      defaultValue: '100',
```

Comment:
@siddarthkay, wouldn't be better to calculate the size in bytes instead and pass `-b` to `du`? Otherwise there's a chance to hit rounding differences depending on the how they calculate the APK size.

Reply:
Indeed `-b` would be more accurate/precise

Reply:
its sad that `du` on MacOS is different and does not have the `-b` flag.
Anyway that shouldn't be a problem since this calculation would be done on CI which is a linux host.

Reply:
https://github.com/status-im/status-jenkins-lib/pull/94/commits/e97837713689924e62d41321bce5e756ee3a748e